concent-builder/jenkins-configuration-files/config.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version='1.1' encoding='UTF-8'?>
```

Comment:
Please move `concent-builder/jenkins-configuration-files/` to `concent-builder/templates/jenkins/` or `concent-builder/files/jenkins/` (depending whether you use templates or static files).